src/core/utils/getResponse.ts

```diff
@@ -9,23 +9,18 @@ export interface ResponseLookupResult {
   response?: Response
 }
 
-export interface ResponseResolutionContext {
-  baseUrl?: string
-}
```

Comment:
We don't have any code that tests for something inclusive of this property, and I couldn't locate anything that defines it, so I'm assuming this became unused during the v1 -> 2 migration?

Reply:
To be safe here, I recommend checking out the v1 tag in Git and taking a peek at what this option was used before, if at all.

Reply:
ooo - it looks like this is used here: https://github.com/mswjs/http-middleware/blob/dfd8c682696f47f3242c6f5a33ab9077a656c5b5/src/middleware.ts#L49C1-L55C11

which i came across, in which case maybe we need to do better at documenting that internally, even in test scenarios, to describe how it works and under what scenarios

Reply:
Agree. A documentation issue.